src/components/Amount.tsx

```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
 
   return (
-    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4">
+    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4 ">
```

Comment:
tile spacing is wrong (left code, right design), should have more space between grid elements

<img width="2345" alt="image" src="https://github.com/user-attachments/assets/e101a8f2-a9f2-40ec-9748-b6432837a7a0">